nmap.go

```diff
@@ -94,6 +98,55 @@ func (s *Scanner) Streamer(stream io.Writer) *Scanner {
 	return s
 }
 
+// Version returns the version of the installed nmap binary
```

Comment:
```suggestion
// Version returns the version of the used nmap binary.
```

This is more correct, since multiple nmap binaries can be installed, but only one is in use by the library at any given time.